src/MetricsProvider.tsx

```diff
@@ -2,6 +2,7 @@ import {
   useLifecycleEventListener,
   useNavigationListener,
 } from '@shopify/react-performance';
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Instead of disabling rules at will (I did it myself in some of our projects), maybe we should [define a config](https://typescript-eslint.io/rules/naming-convention/#options) to the rule in our eslint config to allow `PascalCase` namings?

Reply:
We have just released [eslint-config v.2.1.1](https://github.com/cabify/eslint-config/releases/tag/v2.1.1) version that will fix this issue.